src/app/modules/domain/learning-objects/question/question.component.ts

```diff
@@ -17,14 +18,19 @@ export class QuestionComponent implements OnInit, LearningObjectComponent {
   learningObject: Question;
   checked: boolean[];
   evaluation: MrqEvaluation;
+  private kcId: number;
 
   constructor(private questionService: QuestionService,
-              private instructor: InterfacingInstructor) {
+              private instructor: InterfacingInstructor,
+              private route: ActivatedRoute) {
```

Comment:
I'm not a fan of having to pass multiple services so that one can do it's job. Is it not possible for the InterfacingInstructor to inject activated route and read the kcId when it needs it?